app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizardBody.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
```

Comment:
Let's convert this to the following for now:

```
import React from 'react';
import PropTypes from 'prop-types';
import ModalWizard from '../../components/ModalWizard';
import componentRegistry from '../../../../../../components/componentRegistry';
import MappingWizardGeneralStep from '../MappingWizard/components/MappingWizardGeneralStep';

// TODO remove these, they are space fillers
const t = str => (
  <div align="center">
    <h1>TODO: {str}!</h1>
  </div>
);
const todo = str => (
  <div>
    {t(str)}
    {t(str)}
    {t(str)}
    {t(str)}
    {t(str)}
  </div>
);

class MappingWizardBody extends React.Component {
  shouldComponentUpdate(nextProps, nextState) {
    return (
      JSON.stringify(this.props) !== JSON.stringify(nextProps) ||
      JSON.stringify(this.state) !== JSON.stringify(nextState)
    );
  }
  render() {
    const mappingWizardClustersStepContainer = componentRegistry.markup(
      'MappingWizardClustersStepContainer'
    );
    return (
      <ModalWizard.Body
        {...this.props}
        loadingTitle={__('Loading Infrastructure Mappings...')}
        loadingMessage={__('This may take a minute.')}
        steps={[
          {
            title: __('General'),
            render: () => <MappingWizardGeneralStep />,
            onClick: () => console.log('on step 1 click')
          },
          {
            title: __('Clusters'),
            render: () => mappingWizardClustersStepContainer,
            onClick: () => console.log('on step 2 click')
          },
          {
            title: __('Datastores'),
            render: () => todo('Datastore Mappings Form'),
            onClick: () => console.log('on step 3 click')
          },
          {
            title: __('Networks'),
            render: () => todo('Network Mappings Form'),
            onClick: () => console.log('on step 4 click')
          },
          {
            title: __('Results'),
            render: () => todo('Display Progress and Results'),
            onClick: () => console.log('on step 5 click')
          }
        ]}
      />
    );
  }
}

MappingWizardBody.propTypes = {
  loaded: PropTypes.bool,
  activeStepIndex: PropTypes.number,
  activeStep: PropTypes.string
};

MappingWizardBody.defaultProps = {
  loaded: false,
  activeStepIndex: 0,
  activeStep: '1'
};

export default MappingWizardBody;

```

Reply:
Good find! Fixed in 8e95bcc